tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/api/OpenPaasRestClient.java

```diff
@@ -85,6 +85,18 @@ public Mono<MailAddress> retrieveMailAddress(String openPaasUserId) {
             .onErrorResume(e -> Mono.error(new OpenPaasRestClientException("Failed to retrieve user mail using OpenPaas id " + openPaasUserId, e)));
     }
 
+    public Mono<UserSearchResponse> searchOpenPaasUser(String email) {
```

Comment:
What does it adds compare to `searchOpenPaasUserId` ? 

This method shall not be needed.

Reply:
We need userId+userEmail to create the DavUser object. Otherwise we need to make 2 API calls.